crates/ruma-identity-service-api/src/association/unbind_3pid/v2.rs

```diff
@@ -0,0 +1,68 @@
+//! [POST /_matrix/identity/v2/3pid/unbind](https://matrix.org/docs/spec/identity_service/r0.3.0#post-matrix-identity-v2-3pid-unbind)
+
+use ruma_api::ruma_api;
+use ruma_common::thirdparty::Medium;
+use ruma_identifiers::user_id::UserId;
+use serde::{Deserialize, Serialize};
+
+ruma_api! {
+    metadata: {
+        description: "Remove an association between a session and a Matrix user ID.",
+        method: POST,
+        name: "unbind_3pid",
+        path: "/_matrix/identity/v2/3pid/unbind",
+        rate_limited: false,
+        authentication: AccessToken,
+    }
+
+    request: {
+        /// The Session ID generated by the `requestToken` call.
+        #[serde(skip_serializing_if = "Option::is_none")]
+        pub sid: Option<&'a str>,
+
+        /// The client secret passed to the `requestToken` call.
+        #[serde(skip_serializing_if = "Option::is_none")]
+        pub client_secret: Option<&'a str>,
+
+        /// The Matrix user ID to remove from the 3PIDs.
+        pub mxid: &'a UserId,
+
+        /// The 3PID to remove. Must match the 3PID used to generate the session if using `sid` and
+        /// `client_secret` to authenticate this request.
+        pub threepid:  &'a ThirdPartyId,
+    }
+
+    #[derive(Default)]
+    response: {
+    }
+
```

Comment:
Please keep formatting consistent with the rest of the code:

```suggestion
    response: {}
```